flow/embedded_views.h

```diff
@@ -42,8 +56,88 @@ class ExternalViewEmbedder {
   virtual ~ExternalViewEmbedder() = default;
 
   FML_DISALLOW_COPY_AND_ASSIGN(ExternalViewEmbedder);
+
+#pragma mark - transforms
+  std::shared_ptr<FlutterEmbededViewTransformStack> transformStack;
+
+};  // ExternalViewEmbedder
+
+class FlutterEmbededViewTransformStack {
```

Comment:
Our language is a little ambiguous around "transforms", e.g usually we mean the transform matrix when we say transforms. I don't have a good idea though.

Update: I asked @chinmaygarde and he suggested to call it mutators, I think it's a much more reasonable name.

Reply:
done.